pilot/pkg/config/kube/ior/route.go

```diff
@@ -50,38 +50,18 @@ const (
 	gatewayNamespaceLabel       = maistraPrefix + "gateway-namespace"
 	gatewayResourceVersionLabel = maistraPrefix + "gateway-resourceVersion"
 	ShouldManageRouteAnnotation = maistraPrefix + "manageRoute"
-
-	eventDuplicatedMessage = "event UPDATE arrived but resourceVersions are the same - ignoring"
 )
 
-type syncRoutes struct {
-	metadata config.Meta
-	gateway  *networking.Gateway
-	routes   []*v1.Route
-}
-
 // route manages the integration between Istio Gateways and OpenShift Routes
 type route struct {
```

Comment:
Can you rename this to `routeController`? This will help understanding the code in the rest of the file and remove the variable shadowing in https://github.com/maistra/istio/pull/673/files#diff-d92eadb815953403f29e9eb8198b9b337121ecf2e8571c8f5d3bf0127f3a4353R446.